### DISCUSSION TITLE
Debugging Help Wanted: Does anyone have an 11" M1 iPad Pro with iPad OS 16.4?

### DISCUSSION THREAD

**Discussion by keiwando** [keiwando/toonsquid#267] [General]
There is one very strange bug report and I would like to see if everybody with the same hardware configuration is affected by it or not.

If you have an 11 inch M1 iPad Pro running iPad OS 16.4 (or 16.4.1), please let me know if the following steps work without issues for you:

1. Open a project (can also be a new project)
2. Perform some edits, for example using the path tool and transform tool
3. Close the project (don't quit the app)
4. Open the project again and try to continue editing with the path tool, the transform tool and the brush tool

Do you experience anything unexpected after the second time opening the project?

*(The reported bug causes touch inputs on the canvas to be registered with an offset after the second time of opening the project, making it practically impossible to edit)*

**Comment by sasislm2** [comment]
Yes, it became unresponsive after I reopened the project. M1 iPad Air 16.4.1

**Reply by keiwando** [reply]
Thanks for the info!

**Reply by keiwando** [reply]
@sasislm2 I am unable to reproduce this problem with the M1 iPad Air running 16.4.1. Does this happen with all of your projects and could you please send me a short video of what exactly you are seeing? You can upload it here or send it to contact@toonsquid.com

**Reply by sasislm2** [reply]
It only happens when display settings are set to more space.

**Reply by keiwando** [reply]
Thank you very much! I am able to reproduce it this way.

**Comment by keiwando** [comment]
This issue was fixed in the 1.8.5 update
